app/app/utils.py

```diff
@@ -242,7 +242,8 @@ def sync_profile(handle, user=None, hide_profile=True):
 
         profile.keywords = keywords
         profile.save()
-
```

Comment:
@danlipert shouldnt we add a try catch around
```
profile.organizations =
```
??

if we dont, then none of the code between that and profile.save() will be called, and the profile will not sync!

Reply:
Its already wrapped in a try catch, right? Just no good exception handling. As far as profile.save() is concerned, if getting the orgs fails, then I think the fallout as it stands is that the keywords won't save, but everything else should be saved correctly - assuming we pass on the exception.